lib/ssl/doc/src/using_ssl.xml

```diff
@@ -98,8 +103,10 @@ connection</p>
       <p><em>Step 4:</em> Start the client side: </p>
       <code type="erl">1 client> ssl:start().
 ok</code>
-      <p> To run DTLS add the option {protocol, dtls} to third argument.</p>
-      <code type="erl">2 client> {ok, Socket} = ssl:connect("localhost", 9999,  [], infinity).
+      <p> Be sure to configure certificates to use for server certificate verification.</p>
```

Comment:
Be more precise for learners benefit?

_Be sure to configure certificates to use for_
to 
_Be sure to configure CA certificates to use for_?

Reply:
Reformulated!